vectordatasource/transform.py

```diff
@@ -9329,6 +9331,13 @@ def unpack_viewpoint_claims(shape, props, fid, zoom):
             for viewpoint in _list_of_countries(recognized_by):
                 props['kind:' + viewpoint] = claimed_kind
 
+    if kind and kind.startswith(prefix) and disputed_by:
+
+        # TODO what do I need to do with geometry here?
+        if disputed_by:
+            for viewpoint in _list_of_countries(disputed_by):
+                props['kind:' + viewpoint] = 'unrecognized_disputed_reference_line'
```

Comment:
Modulo other PR comments and IRL discussion update `unrecognized_disputed_reference_line` to `unrecognized_disputed`?